examples/linux_general/providers.tf

```
@@ -0,0 +1,7 @@
+provider "zabbix" {
+  user       = var.user
+  password   = var.password
+  server_url = "http://localhost/api_jsonrpc.php"
```

Please use a variable to allow overriding it without modifying the code
```suggestion
  server_url = var.server_url
```

```
terraform plan -var user=Admin -var password=zabbix -var server_url=http://localhost:8081/api_jsonrpc.php
```

---

Resolved by https://github.com/claranet/terraform-provider-zabbix/pull/7/commits/c143a6ce8df8f41165fa0c5f2269d3cf5c218b19